apps/chat-e2e/src/tests/selectPromptPanelEntity.test.ts

```diff
@@ -36,55 +36,65 @@ dialTest(
     setTestIds('EPMRTC-3667', 'EPMRTC-3669', 'EPMRTC-3666');
     let nestedFolders: FolderInterface[];
     let nestedPrompts: Prompt[] = [];
-    let rootFolder: FolderPrompt;
+    let folderWithPrompts: FolderPrompt;
     let singlePrompt: Prompt;
     let theme: string;
-    let expectedCheckboxColor: string;
-    let expectedEntityBackgroundColor: string;
+    const emptyFolderName = ExpectedConstants.newPromptFolderWithIndexTitle(1);
+    await localStorageManager.setPromptCollapsedSection('Organization');
```

Comment:
let's intro enum for collapsed section values

Reply:
done